docs/src/main/paradox/couchbase.md

```diff
@@ -1,13 +1,23 @@
-#Couchbase
+# Couchbase
 
-The Couchbase connector allows you to read and write to Couchbase. You can query a bucket from CouchbaseSource using N1QL queries or use bulk operation to insert or update with CouchbaseFlow or CouchbaseSink. Couchbase connector uses Couchbase Java SDK behind the scene. 
+@@@ note { title="Couchbase"}
 
-You can read more about Couchbase [here](https://www.couchbase.com/products/server) and you can read more about Couchbase Java SDK [here](https://developer.couchbase.com/documentation/server/current/sdk/java/start-using-sdk.html)
+Couchbase is an open-source, distributed (shared-nothing architecture) multi-model NoSQL document-oriented database software package that is optimized for interactive applications. These applications may serve many concurrent users by creating, storing, retrieving, aggregating, manipulating and presenting data. In support of these kinds of application needs, Couchbase Server is designed to provide easy-to-scale key-value or JSON document access with low latency and high sustained throughput. It is designed to be clustered from a single machine to very large-scale deployments spanning many machines. 
+
+Couchbase provides client protocol compatibility with memcached, but adds disk persistence, data replication, live cluster reconfiguration, rebalancing and multitenancy with data partitioning. 
+
+-- [Wikipedia](https://en.wikipedia.org/wiki/Couchbase_Server)
+
+@@@
+
+Alpakka Couchbase allows you to read and write to Couchbase. You can query a bucket from CouchbaseSource using N1QL queries or reading by document ID. Couchbase connector uses [Couchbase Java SDK verison ${couchbase.version}](https://developer.couchbase.com/documentation/server/current/sdk/java/start-using-sdk.html) behind the scene. 
```

Comment:
The variable substitution in links unfortunately currently does not work: https://github.com/lightbend/paradox/issues/178

Reply:
Actually it does within link labels, since they can contain nested markdown and variable substitution is hence parsed correctly, it just doesn't work within the URLs. So `[Couchbase Java SDK version $couchbase.version$](...)` would have worked.